tests/components/hassio/test_handler.py

```diff
@@ -30,26 +30,64 @@ async def test_api_ping_exeption(hassio_handler, aioclient_mock):
     assert aioclient_mock.call_count == 1
 
 
-async def test_api_homeassistant_info(hassio_handler, aioclient_mock):
-    """Test setup with API Home Assistant info."""
+async def test_api_info(hassio_handler, aioclient_mock):
+    """Test setup with API generic info."""
     aioclient_mock.get(
-        "http://127.0.0.1/homeassistant/info",
-        json={"result": "ok", "data": {"last_version": "10.0"}},
+        "http://127.0.0.1/info",
+        json={
+            "result": "ok",
+            "data": {"supervisor": "222", "homeassistant": "0.110.0", "hassos": None},
+        },
+    )
+
+    data = await hassio_handler.get_info()
+    assert aioclient_mock.call_count == 1
+    assert data["hassos"] is None
+    assert data["homeassistant"] == "0.110.0"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)